homeassistant/components/demo/translations/en.json

```diff
@@ -15,7 +27,8 @@
             "fix_flow": {
                 "abort": {
                     "not_tea_time": "Can not re-heat the tea at this time"
-                }
+                },
+                "step": {}
```

Comment:
What's inserting this empty dict? Is it required?

Reply:
I dunno? I just ran our translation script. It is a generated file, so I didn't put much attention to it otherwise.

Reply:
Yes:

![image](https://user-images.githubusercontent.com/195327/205034049-5cba9d19-ed12-4dd2-b6ac-9f6c91f7c524.png)